fedimint-server/src/consensus/server.rs

```diff
@@ -744,8 +734,11 @@ impl ConsensusServer {
 }
 
 pub(crate) async fn get_finished_session_count_static(dbtx: &mut DatabaseTransaction<'_>) -> u64 {
-    dbtx.get_value(&SignedSessionOutcomeCountKey)
+    dbtx.find_by_prefix_sorted_descending(&SignedSessionOutcomePrefix)
+        .await
+        .next()
         .await
+        .map(|entry| (entry.0 .0) + 1)
```

Comment:
```suggestion
        .map(|entry| (entry.0.0) + 1)
```
that space really messed up my brain

Reply:
Yes, but cargo fmt insists on it.

Reply:
While better than counting, I think the underlying implementation will race ahead and start reading more than just one key, making this call actually generate a heavier IO load than expected.

Reply:
Rocksdb should be caching pages that have been recently written to. This entry is written every session (~5mins) right? Hard to know the actual IO cost without benchmarking it at all but if its significant IO that would be surprising to me

Reply:
I would slap in-memory cache on top of the whole thing, so this code doesn't even need to do any IO except a first time each session.

Reply:
Let's see how it behaves.